src/main/io/osd.c

```diff
@@ -470,15 +487,15 @@ static void osdDrawSingleElement(uint8_t item)
 
             if (isAirmodeActive()) {
                 p = "AIR";
-            }
+		}
```

Comment:
There's still tabs in your change - no idea why this was changed at all, check your editor settings.

Reply:
Hopefully got them all (using the online editor).